models/classes/class.QtiTestService.php

```diff
@@ -570,9 +570,15 @@ protected function importTest(core_kernel_classes_Class $targetClass, Resource $
                 // If any, assessmentSectionRefs will be resolved and included as part of the main test definition.
                 $testDefinition->includeAssessmentSectionRefs(true);
 
-
                 if ($overwriteTest) {
-                    $this->deleteTestsFromClassByTitle($testDefinition->getDocumentComponent()->getTitle(), $testClass, $itemClass);
+                    $testLabel = $testDefinition->getDocumentComponent()->getTitle();
+                    /** @var oat\taoQtiItem\model\qti\metadata\simple\SimpleMetadataValue $m */
+                    foreach($reportCtx->testMetadata as $singleMetadata) {
+                        if(($singleMetadata->getPath()[1] ?? '') === RDFS_LABEL) {
```

Comment:
If I'm not mistaken, there's no need to wrap 
```suggestion
                        if ($singleMetadata->getPath()[1] ?? '' === RDFS_LABEL) {
```